README.md

```diff
@@ -30,15 +30,54 @@ r'                                                      )4XXWW
 .
 ```
 
-## Setup
+## Ruby Setup
+In order to use Roadrunner Rails, you'll need Ruby installed, and either rvm or rbenv to manage your Ruby versions.
+If you don't, follow the directions below:
+
+Install rbenv, which is a tool that helps install/manage versions of Ruby (Note: make sure and follow the brew's post-install instructions):
+
+```
+$ brew install rbenv
+```
+
+And follow the initialization instructions for rbenv, provided by brew
+
+```
+$ rbenv init
+```
+
+Using rbenv install ruby:
+
+```
+$ rbenv install 2.3.0
+```
+
```

Comment:
@plusjeff just added that in ./bin/setup here:

https://github.com/department-of-veterans-affairs/roadrunner-rails/commit/9597772293f7e025cf7010d456b915b46b6e5750

that work?